bindings/python/src/pgvecto_rs/sdk/client.py

```diff
@@ -0,0 +1,130 @@
+from typing import List, Literal, Optional, Tuple, Type, Union
+from uuid import UUID
+
+from numpy import ndarray
+from sqlalchemy import (
+    ColumnElement,
+    Float,
+    create_engine,
+    delete,
+    insert,
+    select,
+    text,
+)
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.engine import Engine
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm.session import Session
+
+from pgvecto_rs.sqlalchemy import Vector
+
+from .filters import Filter
+from .record import Record, RecordORM, RecordORMType
+
+
+class PGVectoRs:
+    _engine: Engine
+    _table: Type[RecordORM]
+    dimension: int
+
+    def __init__(
+        self,
+        db_url: str,
+        collection_name: str,
+        dimension: int,
+    ) -> None:
+        """Connect to an existing table or create a new empty one.
+
+        Args:
+            db_url (str): url to the database.
+            table_name (str): name of the table.
+            dimension (int): dimension of the embeddings.
+        """
+
+        class _Table(RecordORM):
+            __tablename__ = f"collection_{collection_name}"
+            id: Mapped[UUID] = mapped_column(
+                postgresql.UUID(as_uuid=True), primary_key=True
+            )
+            document: Mapped[dict] = mapped_column(postgresql.JSONB)
+            embedding: Mapped[ndarray] = mapped_column(Vector(dimension))
+
+        self._engine = create_engine(db_url)
+        with Session(self._engine) as session:
+            session.execute(text("CREATE EXTENSION IF NOT EXISTS vectors"))
+            session.commit()
+        self._table = _Table
+        self._table.__table__.create(self._engine)
+        self.dimension = dimension
+
+    def upsert(self, records: List[Record]) -> None:
+        with Session(self._engine) as session:
+            for record in records:
+                session.execute(
+                    insert(self._table).values(
```

Comment:
What does "upsert" here mean? Will the ORM handle the id collision and update the record?